armory/scenarios/base.py

```diff
@@ -139,22 +174,24 @@ def run_config(config_json, from_file=False):
     )
     parser.add_argument(
         "--no-docker",
-        dest="no_docker",
-        action="store_const",
-        const=True,
-        default=False,
+        action="store_true",
         help="Whether to use Docker or a local environment with armory run",
     )
     parser.add_argument(
         "--load-config-from-file",
         dest="from_file",
-        action="store_const",
-        const=True,
-        default=False,
+        action="store_true",
         help="If the config argument is a path instead of serialized JSON",
     )
+    parser.add_argument(
+        "--mongo",
+        dest="mongo_host",
+        default=None,
+        help="Send scenario results to a MongoDB instance at the given host",
```

Comment:
Could we add an example of what a Mongo host URI with username/password looks like so it's clear how to use credentials. Alternatively we could add some notes in the documentation.

Reply:
I pushed an update to this text providing examples of how to specify the host. The code as currently written to connect to the host assumes that the MongoDB instance does not use authentication and thus does not handle usernames or passwords, which I can document either here or elsewhere, as you prefer

Reply:
Apologies; I just reread https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html#pymongo.mongo_client.MongoClient and see that the code as written can take a username and password